### PR TITLE
Update bug report template

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Before proceeding, please update to latest version and test if the issue persist
 ## Describe the bug in a sentence or two.
```
Comment (strausr) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Title? 
Something like Bug report for React SDK?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Before proceeding, please update to latest version and test if the issue persist
...
 …
 
+## Describe the desired/expected behavior.
+…
+
```
Comment (strausr) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: I would also add a section for reproduction steps and a section for versions used in core and react:)

**Review by strausr — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (maoznir) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: already exists, but updated it, please see last commit

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (maoznir) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: Already exists

**Review by strausr — APPROVED** [review]

**Review by patrick-tolosa — APPROVED** [review]

**Pull request merged**